afb-binding/src/verbs-frontend.rs

```diff
@@ -374,20 +374,26 @@ fn authorize_response(
         _ => return afb_error!("ocpp-authorize-start", "invalid response type"),
     };
 
+    let ocpp_auth_state;
     match response.id_tag_info.status {
         v106::AuthorizationStatus::Accepted => {
             afb_log_msg!(Debug, rqt, "ocpp-authorize-done");
-            ctx.mgr.authorized(true)?;
+            ocpp_auth_state = ctx.mgr.authorized(true)?;
+        }
+        v106::AuthorizationStatus::Blocked => {
+            afb_log_msg!(Debug, rqt, "ocpp-authorize-blocked");
+            ocpp_auth_state = ctx.mgr.authorized(false)?;
```

Comment:
I would suggest something like:
```
match response.id_tag_info.status {
        v106::AuthorizationStatus::Accepted => {
            afb_log_msg!(Debug, rqt, "ocpp-authorize-done");
            ocpp_auth_state = true;
        }
        v106::AuthorizationStatus::Blocked => {
            afb_log_msg!(Debug, rqt, "ocpp-authorize-blocked");
            ocpp_auth_state = false;
        }
...
}
    ctx.mgr.authorized(ocpp_auth_state)?;
    afb_log_msg!(Notice,None,"OCPP AUTH STATE: {}", ocpp_auth_state);
    rqt.reply(ocpp_auth_state, 0);
}
```

This way you do not need to change the return type of ` authorized`

Reply:
Modifs are done.